lib/Sensors/Gateway.js

```diff
@@ -48,31 +56,53 @@ Gateway.prototype.getData = function (data) {
         obj.proto_version = this.protoVersion;
         newData           = true;
     }
-
-    if ((data.ac_state !== undefined) && (data.ac_state !== this.acState)) {
```

Comment:
This is not needed anymore?

Reply:
Yes, it is no longer needed. ac_state carries little useful information